Add Galactifun (#162)
Slimefun/Wiki: pages/Addons.md

PotionExpansion | EpicPlayerA10 | <details><summary>expand</summary>An addon which adds new potions that give you legal xray!</details> | [Readme](https://github.com/EpicPlayerA10/PotionExpansion/blob/master/README.md) | [GitHub](https://github.com/EpicPlayerA10/PotionExpansion) | [Builds](https://thebusybiscuit.github.io/builds/EpicPlayerA10/PotionExpansion/master/)
FlowerPower | NCBPFluffyBear | <details><summary>expand</summary>FlowerPower is a magic addon centered around experience and flowers. The base of each recipe requires various flowers, and different items are crafted along the way to make mid to endgame items. A majority of these items function by consuming experience in exchange for a certain effect or buff.</details> | [Readme](https://github.com/NCBPFluffyBear/FlowerPower/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/FlowerPower/) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FlowerPower/master/)

> Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inteded to replace the apparently abandoned ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)

Reviewer: ```suggestion
Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon intended to replace the apparently abandoned ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```

Reviewer: gonna mark as resolved as walshy requested changes on the same part

> Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inteded to replace the apparently abandoned ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)

Reviewer 2: ```suggestion
Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inspired by ClayTech. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
```

ClayTech is not "apparently abandoned". It had updated forks and the last commit was 2 weeks ago. This has been stated time and time again and this is just misleading.
